pkg/api/v1/register.go

```diff
@@ -39,7 +39,7 @@ func init() {
 
 // Adds the list of known types to api.Scheme.
 func addKnownTypes() {
-	api.Scheme.AddKnownTypes("v1",
+	api.Scheme.AddKnownTypes(api.Group, "v1",
```

Comment:
We should define a function getGroupForKind() which, for now, can always return api.Group irrespective of the kind.